cves/kernel/CVE-2018-10940.yml

```diff
@@ -235,8 +246,11 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
+  - commit: 853fe1bf7554155376bb3b231112cdff9ff79177
+    note: |
+      This commit highlights that function tables are a common place to look for attackers, so they changed the tables
```

Comment:
This is unlikely related to the original vulnerability as this vulnerability is a read-out-of-bounds making it impossible to write to this vtable.

Reply:
I just figured I would mention it as an interesting-to-me commit, not so much related to the vulnerability.